applications/crossbar/src/crossbar_services.erl

```diff
@@ -82,9 +82,12 @@ check_creditably(Context, _Services, _Quotes, Amount) when Amount =< 0 ->
 check_creditably(Context, Services, _Quotes, Amount) ->
     Options = #{amount => kz_currency:dollars_to_units(Amount)},
     case kz_services:is_good_standing(Services, Options) of
-        'true' -> Context;
-        'false' ->
-            cb_context:add_system_error('no_credit', Context)
+        {'true', _} -> Context;
```

Comment:
Similar spec issue here with `boolean() | number()` I think.

Reply:
But `boolean() | number()` is correct, no?